AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/internal/core/handler/FragmentArgHandler.java

```diff
@@ -106,7 +106,10 @@ private void createBuilderInjectionMethod(Element element, EFragmentHolder holde
 		JClass paramClass = codeModelHelper.typeMirrorToJClass(type);
 
 		JMethod method = builderClass.method(PUBLIC, holder.narrow(builderClass), fieldName);
-		JVar arg = method.param(paramClass, fieldName);
+		JExpression arg = method.param(paramClass, fieldName);
+		if (bundleHelper.isParcelerBean()) {
```

Comment:
Maybe we could just create the whole expression with `BundleHelper` just like when we restore, instead of only get the method name, and leave the caller (AnnotationHandler) to create the whole saving expression.